app/views/docs/realtime.phtml

```diff
@@ -320,13 +320,13 @@ subscription.close()</code></pre>
     <tbody>
         <tr>
             <td data-title="Name: ">
-                event
+                events
             </td>
             <td data-title="Type: ">
-                string
+                string[]
             </td>
             <td data-title="Description: ">
-                The <a href="/docs/webhooks#events">system event</a> that triggered this update.
+                The <a href="/docs/webhooks#events">system events</a> that triggered this update.
```

Comment:
Hmm, do we only have events documented under webhooks? How do people using Functions learn about all the events? This might be a concern for separate task.

Reply:
Yeah, we need to review this. Let me open a separate task for this.